ServerlessBenchmark/PerfResultProviders/FunctionLogs.cs

```diff
@@ -43,25 +37,42 @@ public static bool RemoveAllCLoudWatchLogs(string functionName)
             return isEmpty;
         }
 
-        private static List<AzureFunctionLogs> GetAzureFunctionLogsInternal(string functionName, DateTime? startTime, DateTime? endTime, bool update = false, int expectedExecutionCount = 0, int waitForAllLogsTimeoutInMinutes = 5)
+        private static List<AzureFunctionLogs> GetAzureFunctionLogsInternal(string functionName, DateTime? startTime, int expectedExecutionCount, int waitForAllLogsTimeoutInMinutes, bool includeIncomplete)
         {
             Console.WriteLine("Getting Azure Function logs from Azure Storage Tables..");
             var connectionString = ConfigurationManager.AppSettings["AzureStorageConnectionString"];
+            var _azurefunctionLogs = new List<AzureFunctionLogs>();
 
             if (!string.IsNullOrEmpty(connectionString))
             {
                 var storageAccount = CloudStorageAccount.Parse(connectionString);
                 var tableClient = storageAccount.CreateCloudTableClient();
-                var table = tableClient.GetTableReference("AzureFunctionsLogTable");
-                var query = new TableQuery<AzureFunctionLogs>();
+                var table = tableClient.GetTableReference("AzureFunctionsLogTable");                
                 int size = 0;
                 var latestNewLog = DateTime.UtcNow;
                 var lastSize = 0;
 
                 do
-                {
-                    _azurefunctionLogs = table.ExecuteQuery(query).Where(log => string.IsNullOrEmpty(functionName) || (!string.IsNullOrEmpty(log.FunctionName) &&
-                        log.FunctionName.Equals(functionName, StringComparison.CurrentCultureIgnoreCase) && !string.IsNullOrEmpty(log.ContainerName))).ToList();
+                {                    
+                    var query = table.CreateQuery<AzureFunctionLogs>().Where(x => x.PartitionKey == "R");
```

Comment:
FYI. Recreating table could be a better idea, but it also requires a retry. If you try to create table you've just removed it fails as removing operation is still in progress.